grakn-graph/src/main/java/ai/grakn/graph/internal/ConceptCache.java

```diff
@@ -38,11 +38,11 @@
  * @author fppt
  *
  */
-class ComponentCache<V> {
+class ConceptCache<V> {
```

Comment:
It can cache anything but at this stage it is ONLY used for caching concept data hence the name change. Just to provide a bit more context between `TxCache`, `GraphCache`, and `ConceptCache`